kasa/smartcam/modules/pantilt.py

```diff
@@ -88,9 +89,43 @@ async def set_tilt_step(value: int) -> None:
             )
         )
 
+    async def _post_update_hook(self) -> None:
+        """Update presets after update."""
+        presets_response = await self._device._query_helper(
+            "getPresetConfig", {"preset": {"name": ["preset"]}}
+        )
```

Comment:
There is no need to manually query here, the `query()` will be used automatically.

You can then use the `data` property to access the module data like this:
https://github.com/python-kasa/python-kasa/blob/347bf9a419d20089227cc4b9eaa2af257ef32b82/kasa/smartcam/modules/alarm.py#L110-L113